solr/core/src/java/org/apache/solr/security/PKIAuthenticationPlugin.java

```diff
@@ -374,8 +374,17 @@ PublicKey fetchPublicKeyFromRemote(String nodename) {
     }
   }
 
+  @Override
+  public void setup(Http2SolrClient.Builder httpClientBuilder, Http2SolrClient client) {
+    setup(client, httpClientBuilder);
+  }
+
   @Override
   public void setup(Http2SolrClient client) {
+    setup(client, null);
+  }
+
+  private void setup(Http2SolrClient client, Http2SolrClient.Builder builder) {
```

Comment:
why have this separate and with different parameter order?  In other words, lets make *this* one the `@Override` with builder then client.  Remove the 4 lines earlier on 377.